packages/account/src/Components/trading-assessment/trading-assessment-dropdown.jsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import { Field } from 'formik';
+import { DesktopWrapper, Dropdown, MobileWrapper, Text, SelectNative } from '@deriv/components';
+import { localize } from '@deriv/translations';
+
+const TradingAssessmentDropdown = ({ item_list, onChange, values, setFieldValue, setEnableNextSection }) => {
+    React.useEffect(() => {
+        checkIfAllFieldsFilled();
+    }, [values]);
+
+    const checkIfAllFieldsFilled = () => {
+        if (values) {
+            setEnableNextSection(
+                values.cfd_experience &&
+                    values.cfd_frequency &&
+                    values.trading_experience_financial_instruments &&
+                    values.trading_frequency_financial_instruments
+            );
+        }
+    };
+
+    return (
+        <div className='trading-assessment__wrapper__dropdown'>
+            {item_list.map((question, index) => (
+                <Field name={question.form_control} key={index}>
```

Comment:
using index as a key in anti-pattern due to this https://articles.wesionary.team/using-index-as-a-key-is-an-anti-pattern-in-react-8e5db3aea3a9